### ISSUE TITLE
visionOS crash on launch

### ISSUE THREAD

**Issue by eurobob** [mchakravarty/CodeEditorView#122]
Thread 1: "*** -[NSConcreteTextStorage attribute:atIndex:longestEffectiveRange:inRange:]: Range or index out of bounds"

Happens on simulator and actual device. Works fine on ipad. It's a basic fresh ios App with the example code added

**Comment by mchakravarty** [comment]
Yes, I'm seeing this crash, too. Interestingly, it doesn't happen on visionOS 1.0 (in the simulator), but it starts happening from visionOS 1.1 onwards.

Unfortunately, the crash log only includes methods from system libraries. That does make it trickier to track down.
